Kitodo/src/main/webapp/pages/processFromTemplate.xhtml

```diff
@@ -36,40 +36,55 @@
         <h3 id="headerText">
             <h:outputText value="#{msgs.createNewProcess} (#{msgs.template}: '#{CreateProcessForm.template.title}')"/>
         </h3>
-        <p:commandButton id="save"
-                         value="#{msgs.save}"
-                         action="#{CreateProcessForm.createNewProcess}"
-                         icon="fa fa-floppy-o fa-lg" iconPos="right"
-                         onclick="setConfirmUnload(false);PF('notifications').renderMessage({'summary':'#{msgs.validatingData}','detail':'','severity':'info'});"
-                         update="notifications, sticky-notifications"/>
-        <p:commandButton id="saveContinue"
-                         value="#{msgs.save} + #{msgs.newElement}"
-                         action="#{CreateProcessForm.createNewProcessAndContinue}"
-                         icon="fa fa-floppy-o fa-lg"
-                         iconPos="right"
-                         onclick="setConfirmUnload(false);PF('notifications').renderMessage({'summary':'#{msgs.validatingData}','detail':'','severity':'info'});"
-                         update="notifications, sticky-notifications"
-                         styleClass="secondary"/>
-        <p:commandButton id="opacSearch"
-                         process="@this"
-                         value="#{msgs['newProcess.catalogueSearch.heading']}"
-                         oncomplete="PF('catalogSearchDialog').show();"
-                         icon="fa fa-book"
-                         iconPos="right"
-                         update="catalogSearchForm"
-                         styleClass="secondary">
-            <p:resetInput target="catalogSearchForm"/>
-        </p:commandButton>
-        <p:commandButton id="fileUpload"
-                         process="@this"
-                         value="#{msgs['newProcess.fileUpload.heading']}"
-                         oncomplete="PF('fileUploadDialog').show();"
-                         icon="fa fa-upload"
-                         iconPos="right"
-                         update="fileUploadForm"
-                         styleClass="secondary">
-            <p:resetInput target="fileUploadForm"/>
-        </p:commandButton>
+        <p:splitButton id="saveContinue"
```

Comment:
I would prefer to _not_ combine both `Save` buttons into one split button but leave them as they are as two separate buttons that are always displayed, because they seem both equally relevant to me.

Reply:
It will look then like this

<img width="1427" alt="Bildschirmfoto 2021-11-25 um 13 33 45" src="https://user-images.githubusercontent.com/34158824/143442617-1b6146eb-88c9-426f-a998-b8812ddce06e.png">